Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -578,59 +729,59 @@ public class ConnackPacket {
     }
 
     static func convertFromNative(_ from: UnsafePointer<aws_mqtt5_packet_connack_view>?) -> ConnackPacket? {
+        guard let from = from else {
+            return nil
+        }
+        let connackView = from.pointee
+
+        let sessionPresent = connackView.session_present
+        guard let reasonCode = ConnectReasonCode(rawValue: Int(connackView.reason_code.rawValue))
+        else { fatalError("aws_mqtt5_packet_connack_view from native missing a reason code.") }
+        let sessionExpiryInterval = (connackView.session_expiry_interval?.pointee).map { TimeInterval($0) }
```

Comment:
This can result in nil.map, does that work?

Reply:
`map` works for Optional: https://developer.apple.com/documentation/swift/optional/map(_:)